src/cursor/mod.rs

```diff
@@ -107,9 +109,18 @@ where
                 PinnedConnection::new(pin),
                 provider,
             ),
+            kill_watcher: None,
             _phantom: Default::default(),
         }
     }
+
+    #[allow(dead_code)]
```

Comment:
This (and the equivalent method on `SessionCursor`) trigger a false-positive dead code warning.

Reply:
Done.